lego/apps/events/models.py

```diff
@@ -652,7 +653,10 @@ def total_capacity(self):
     @property
     def registration_count(self):
         """ Prefetch friendly counting of registrations for an event. """
-        return sum([pool.registrations.all().count() for pool in self.pools.all()])
+        return sum(
```

Comment:
nit: The second parameter is the "start value" of the sum, so it might  be easier to just add the numbers together instead.
```suggestion
        return self.legacy_registration_count + sum(
```